### ISSUE TITLE
Save a GridObject as geotiff

### ISSUE THREAD

**Issue by wschwanghart** [TopoToolbox/pytopotoolbox#97]
Hi, is there already a way to save a GridObject as geotiff? I think this could be easily implemented using the rasterio tools.

**Comment by Teschl** [comment]
Yes, the `write_tif()` function is able to create geotiff files from GridObjects.

**Comment by wkearn** [comment]
@wschwanghart: what do you think would have made it easier to discover the `write_tif` function in our pytopotoolbox documentation? 

Would having an example that uses `write_tif` make it more visible? More structure to our [API documentation](https://topotoolbox.github.io/pytopotoolbox/api.html)? A more intuitive name? This could be a good opportunity to think about how our documentation is organized.
